### PR TITLE
Mobile auth native

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **Added new content and edited existing content** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-232602](https://oktainc.atlassian.net/browse/OKTA-232602)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/before-you-begin/android/sample.md [hunk 1]
```diff
@@ -0,0 +1 @@
+If you want to skip this guide and just download a working sample app, download our [Android example](https://github.com/okta/samples-android/tree/master/browser-sign-in).
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/before-you-begin/android/sample.md]: This needs to be: https://github.com/okta/samples-android/tree/master/custom-sign-in

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/before-you-begin/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Before you begin
+---
+You can connect your mobile app to Okta and sign users in by opening a browser. But, if you prefer that your users not leave your app, you need to build a custom sign-in UI with native controls and screens instead. Use this guide to build a customized sign-in experience inside your mobile application.
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/before-you-begin/index.md]: Consider linking:

> sign users in by [opening a browser](https://developer.okta.com/docs/guides/sign-into-mobile-app/android/before-you-begin/).

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/before-you-begin/ios/sample.md [hunk 3]
```diff
@@ -0,0 +1 @@
+If you want to skip this guide and just download a working sample app, download our [iOS example](https://github.com/okta/samples-ios/tree/master/browser-sign-in-and-biometric-storage).
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/before-you-begin/ios/sample.md]: This link needs to be: https://github.com/okta/samples-ios/tree/master/custom-sign-in/

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+title: Add and configure packages
+---
+Next, you need to add Okta to your application by installing and configuring our SDK.
+
+You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/android/configure-packages/) guide.
+
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md]: We need a new para here (and a stack selector) that shows how to install the additional package (Authentication SDK) required for building custom sign-in UI, on top of the package that was already installed by following the previous guide. For example:

> To build custom sign-in UI,  you need to install one additional package: the Okta Authentication SDK. This SDK works together with the OIDC SDK you already have installed to make authentication requests to Okta from your own views.
> <stack selector: ios/android>

Comment (nbarbettini) [reply packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md]: The content in this stack selector should show:
* How to install the package
* How to configure the package/initialize the SDK in code

The first few paras of `Use the Okta Java Authentication SDK` from the primary-authentication step should be moved here as the Android content. @IldarAbdullin-okta can provide the iOS content.

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md]: I added a new paragraph, created the folders for the stack selector, added notes on what the stack selector should show in the iOS file and moved the install okta auth sdk content from the Android primary-authentication step into the new Android file.

Could @FeiChen-okta please check the changes for Android?
@nbarbettini - please verify that the changes are what you wanted.
@IldarAbdullin-okta - can you please provide the iOS content for the install okta auth sdk?

Comment (IldarAbdullin-okta) [reply packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md]: @susanharper-okta , sure, will submit into this branch

Comment (FeiChen-okta) [reply packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md]: @susanharper-okta The android part is correct.

Comment (IldarAbdullin-okta) [reply packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md]: Double checked iOS part. Changes are there

**Pull request merged**